client/components/Search.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+class Search extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchTerm: '',
+    };
+  }
+
+  search(e) {
+    e.preventDefault();
+    let searchTerm = this.state.searchTerm;
+    const fitsFilter = this.props.reviews.filter((review) => {
```

Comment:
What were your trade-offs/thoughts on having search locally scoped to this component vs. at the app-level? It seems like this conveniently keeps the Search Term out of App-level state, at the cost of not being able to filter the videos at this level. I'd also be curious if the only reason you need to pass reviews down is to perform line 14. If so, I'd consider bringing that state back to the App-level